flink/README.md

```diff
@@ -56,11 +56,19 @@ _Available for Agent >6.0_
 1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j.properties`, `log4j-cli.properties`, `log4j-yarn-session.properties`, or `log4j-console.properties` file. See [Flink's repository][6] for default configurations. For example `log4j.properties` contains this configuration by default:
 
    ```conf
-   log4j.appender.file=org.apache.log4j.FileAppender
-   log4j.appender.file.file=${log.file}
-   log4j.appender.file.append=false
-   log4j.appender.file.layout=org.apache.log4j.PatternLayout
-   log4j.appender.file.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p %-60c %x - %m%n
+   appender.main.name = MainAppender
```

Comment:
Overall I'm not sure how much value this excerpt really adds, and it easily becomes out-dated.

Reply:
@zentol sorry about the delay. Do you think it would be possible to instead link to some examples from Flink docs?

Reply:
I've updated the logging section. :+1:

Reply:
Cool! I'll merge once the checks complete! Thanks again!